tests/Feature/ProductFilterTest.php

```diff
@@ -184,9 +189,14 @@ public function testProductMaxPriceWithVatHiddenChannel(string $user): void
         // Default from migration with vat_rate 0
         $defaultChannel = SalesChannel::query()->where('default', '=', true)->first();
 
+        /**
+         * I'm not sure what result should this really return, how is user access to PRIVATE SalesChannel determined?
```

Comment:
The user may have access to a `private` sales channel if that sales channel is assigned to the organization to which the user belongs.

It's possible that this test should be named differently. The test was intended to verify whether the fallback for sales channels works correctly and filters the products properly. In this case, the user does not have access to the hidden channel, so they should see both products on the list if the max price is set to 10, because the VAT from the default channel, which is 0, will apply, not 25 as it is in the hidden channel.